subprojects/gradle/build-checks/src/main/kotlin/com/avito/android/build_checks/AndroidAppChecksExtension.kt

```diff
@@ -38,28 +39,30 @@ public open class AndroidAppChecksExtension : BuildChecksExtension() {
              */
             public val ignoredResourceTypes: MutableList<String> = mutableListOf()
 
+            public val ignoredResources: MutableSet<Resource> = mutableSetOf()
```

Comment:
could it be `internal`, and `com.avito.android.build_checks.internal.unique_app_res.Resource` too then?

Reply:
Thanks, didn't think of it. Fixed